.vscode/settings.json

```diff
@@ -1,9 +1,15 @@
 {
-  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
+  "editor.formatOnPaste": false, // required
+  "editor.formatOnType": false, // required
+  "editor.formatOnSave": true, // optional
+  "editor.formatOnSaveMode": "file", // required to format on save
+  "files.autoSave": "onFocusChange", // optional but recommended
+  "vs-code-prettier-eslint.prettierLast": false, // set as "true" to run 'prettier' last not first
```

Comment:
I added these from my original workspace settings for `prettier-eslint` as I believe they were recommended to add in, let me know and we can modify as needed

Reply:
I was trying to remember where these settings came from. These are actually recommended by the `Prettier ESLint` extension with the exact comments, I would rather keep them if they are recommended for the configuration of the project. Let me know if you feel otherwise, I have these exact settings from the artist-portal and never ran into issues...